cmd/relay.go

```diff
@@ -104,3 +105,10 @@ func bindBootnodeFlag(flags *pflag.FlagSet, config *relay.Config) {
 	flags.IntVar(&config.MaxResPerPeer, "max-reservations", 512, "Updates max circuit reservations per peer (each valid for 30min)") // TODO(corver): Align flag name to p2p-max-reservations
 	flags.IntVar(&config.MaxConns, "p2p-max-connections", 16384, "Libp2p maximum number of peers that can connect to this bootnode.")
 }
+
+// bindP2PFlagsRelay binds p2p flags for the relay command.
+func bindP2PFlagsRelay(cmd *cobra.Command, config *p2p.Config) {
```

Comment:
can't remove `p2p-udp-address` yet since it needs to support it for older clusters.